CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Fix linkage of Boost.Serialization on Windows ([#2400](https://github.com/stack-of-tasks/pinocchio/pull/2400))
 - Fix mjcf parser appending of inertias at root joint ([#2403](https://github.com/stack-of-tasks/pinocchio/pull/2403))
 
+### Changed
+- Change parsing of model (urdf, mjcf and sdf) when no root joint is provided and let users decide of root joint name ([#2402](https://github.com/stack-of-tasks/pinocchio/pull/2402))
```

Comment:
You have added methods to allow to change the root joint name.

You can modify this line and move it in the added section.